files/image_config/copp/copp_cfg_chassis.j2

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
do we need to create a new file for chassis? Can we update the existing j2  file  to modify the queue for voq switch-type?